Xamarin.Forms.Core/TapGestureRecognizer.cs

```diff
@@ -34,18 +34,16 @@ public int NumberOfTapsRequired
 			set { SetValue(NumberOfTapsRequiredProperty, value); }
 		}
 
-		public event EventHandler Tapped;
+		public event TappedEventHandler Tapped;
 
 		[EditorBrowsable(EditorBrowsableState.Never)]
-		public void SendTapped(View sender)
+		public void SendTapped(View sender, Point position)
```

Comment:
breaking change

Reply:
Add a new Method and call it SendTappedWithPosition